components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.probes/src/main/java/org/wso2/micro/integrator/probes/ReadinessResource.java

```diff
@@ -96,6 +100,19 @@ public boolean invoke(MessageContext synCtx) {
         }
 
         axisCtx.setProperty(HTTP_SC, CACHED_RESPONSE_CODE);
+        if (HOT_DEPLOYMENT == null) {
+            Parameter hotDeployment = axisCtx.getConfigurationContext().getAxisConfiguration().getParameter(
+                    TAG_HOT_DEPLOYMENT);
+            if (hotDeployment != null) {
+                HOT_DEPLOYMENT = JavaUtils.isTrue(hotDeployment.getValue(), false);
+            }
+        }
+
+        if (HOT_DEPLOYMENT) {
+            log.warn("Hot Deployment and Readiness Probe configurations are both enabled in your server! Note that " +
+                             "the readiness probe will not identify faulty artifacts that are hot deployed. Be sure " +
+                             "to disable hot deployment if the readiness probe is enabled.");
```

Comment:
The + sign need to come in the start . Shall we format this.

Reply:
Done.